smalltalksrc/CAST/CASTParserTests.class.st

```diff
@@ -686,10 +694,11 @@ CASTParserTests >> testParseSequenceOfDeclarators [
 { #category : #'tests-declarators' }
 CASTParserTests >> testParseSimpleArrayDeclaration [
 
-	| declaration declarator |
+	| declaration declarator declarationSpecifiers |
 	declaration := self parseDeclaration: 'int a[]'.
-	
-	self assert: (declaration specifiers includes: 'int').
+	declarationSpecifiers := declaration specifiers.
+
+	self assert: declarationSpecifiers first value equals: 'int'.
```

Comment:
Maybe we needed some `self assertIntDeclaration:` to avoid repeating the same change :P